Android/app/src/main/java/com/example/boardgamesocial/MainActivity.java

```diff
@@ -2,13 +2,46 @@
 
 import androidx.appcompat.app.AppCompatActivity;
 
+import android.content.Context;
+import android.content.Intent;
 import android.os.Bundle;
+import android.widget.Button;
+import android.widget.EditText;
 
 public class MainActivity extends AppCompatActivity {
 
+    private EditText etUsername, etPassword;
+    private Button btnLogin, btnSignup;
+
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
+
+        wireUpDisplay();
+    }
+
+    private void wireUpDisplay() {
+
+        etUsername = findViewById(R.id.et_login_username);
+        etPassword = findViewById(R.id.et_login_password);
+        btnLogin = findViewById(R.id.btn_login);
+        btnSignup = findViewById(R.id.btn_signup);
+
+        btnLogin.setOnClickListener(view -> {
+            //TODO: 1) Verify user based on credentials
```

Comment:
Similar to sign up if 400 then you send wrong credential. Otherwise will return token and status 200